packages/wallet/backend/src/app.ts

```diff
@@ -344,4 +344,9 @@ export class App {
     process.nextTick(() => this.processPendingTransactions())
     process.nextTick(() => this.processWMWalletAddresses())
   }
+
+  async createDefaultUsers() {
+    const userService = this.container.resolve('userService')
+    await userService.createDefaultAccount()
```

Comment:
will be useful to have a message when the default accounts are created

Reply:
@dragosp1011 are you thinking a message for the user? I was thinking of updating README and explaining there for local dev people. Would that be enough?

Reply:
a log in backend app to say that default accounts are ready will be helpful to know when they can be used